app/dao/__init__.py

```diff
@@ -17,13 +17,6 @@ def create_instance(self, inst, _commit=True):
         if _commit:
             db.session.commit()
 
-    def update_instance(self, inst, update_dict, _commit=True):
-        # Make sure the id is not included in the update_dict
-        update_dict.pop("id")
-        self.Meta.model.query.filter_by(id=inst.id).update(update_dict)
-        if _commit:
-            db.session.commit()
-
```

Comment:
I'm not sure I follow why this method was removed; wouldn't this still be needed?

Reply:
It's never used anywhere.

Reply:
Sure enough it's not!